apps/mobile/src/features/ImportPrivateKey/ImportPrivateKey.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { render, screen, fireEvent, act, waitFor } from '@/src/tests/test-utils'
+import { ImportPrivateKey } from './ImportPrivateKey.container'
+
+jest.mock('@/src/navigation/useScrollableHeader', () => ({
+  useScrollableHeader: () => ({
+    handleScroll: jest.fn(),
+  }),
+}))
+
+describe('ImportPrivateKey', () => {
+  it('renders the import private key screen', () => {
+    render(<ImportPrivateKey />)
+
+    expect(screen.getByText('Import a private key')).toBeTruthy()
+    expect(
+      screen.getByText(
+        'Enter your seed phrase or a private key below. Make sure to do so in a safe and private place.',
+      ),
+    ).toBeTruthy()
+  })
+
+  it('enables import button when private key is entered', async () => {
+    render(<ImportPrivateKey />)
+
+    const input = screen.getByPlaceholderText('Paste here or type...')
+    const button = screen.getByText('Import signer')
+
+    act(() => fireEvent.press(button))
+
+    await waitFor(() => {
+      expect(screen.getByTestId('safe-input').props.style.borderTopColor).toBe('#FF5F72')
```

Comment:
I don't like this. I see that we did the same in the SafeWalletInput PR, but the problem is that the moment we go for a different border color those tests will fail. I think that instead we should read the color variable in tamagui here.

Reply:
I'll use the input theme reference here 👍